asreview/io/xml_reader.py

```diff
@@ -0,0 +1,119 @@
+import xml.etree.ElementTree as ET
+
+import pandas as pd
+
+from asreview.io.utils import _standardize_dataframe
+
+
+class EndnoteXMLReader:
+    """Endnote XML file reader."""
+
+    read_format = [".xml"]
+    write_format = [".csv", ".tsv", ".xlsx"]
```

Comment:
```suggestion
    write_format = [".csv", ".tsv", ".xlsx", ".ris"]
```

@terrymyc what does it imply to add RIS here?